.github/workflows/smoke-test-daily.yml

```diff
@@ -45,12 +46,6 @@ jobs:
               working-directory: plugins/woocommerce
               run: pnpm exec playwright install chromium
 
-            - name: Run 'Update WooCommerce' test.
```

Comment:
Took this out from here because we're going to be deleting the plugin and reinstalling it during setup.